### DISCUSSION TITLE
Add option to grey out ''Move Set'' when block is active

### DISCUSSION THREAD

**Discussion by iliadaili** [proginosko/LeechBlockNG#401] [Ideas]
Hello!

The move set option is something that I've asked for some time ago and it is absolutely incredible, helps a lot and I use it constantly. It did create a problem though, that being the fact that, even when I prevent access to options for a block when the block is active, I can still bypass that by moving another block in front of it and editing it to, for example, have the same sites that the locked one does, but filtering them with grayscale instead of blocking them, and it will take precedence, since it is ahead of the other block. Of course, I can prevent this by locking the leechblock options page entirely, but that and any other way to prevent this bypass ends up preventing me from editing other block lists, editing general options, etc. For that reason, I believe that having an option to grey out the ''Move Set'' button when a certain block is active can be very efficient in eliminating any bypassing issues that can come with that feature, without restricting the user's access to other things. Thanks for the attention spent and thank you for the work you put into this extension.

**Comment by proginosko** [comment]
I think that would require disabling _all_ the "Move Set" buttons, since in principle any set could be moved in front of the active set. Right?

**Reply by iliadaili** [reply]
Yes, that's what I was thinking. The ''move set'' option would not exist while a block that has this option that I'm suggesting checked is active, to give the users an option to prevent bypassing locked blocks without having to lock themselves out of Leechblock's options page. Sorry for taking such a long time to respond, but yes, this is what I was thinking.

**Reply by proginosko** [reply]
Actually, on reflection, it only requires (1) disabling the move buttons for the disabled set, (2) disabling the move-right button for the set _before_ the disabled set (if there is one), and (3) disabling the move-left button for the set _after_ the disabled set (if there is one).

This will be implemented in the next release.

**Reply by iliadaili** [reply]
Thank you!
